module/programUsers/helper.js

```diff
@@ -96,22 +96,23 @@ module.exports = class ProgramUsersHelper {
                 if (fieldsArray != "all") {
                     fieldsArray.forEach(field => {
                         projection[field] = 1;
-                   });
-               }
+                    });
+                }
                
-               if( skipFields !== "none" ) {
+                if( skipFields !== "none" ) {
                    skipFields.forEach(field=>{
                        projection[field] = 0;
                    });
-               }
-               
-               let programUsers = 
-               await database.models.programUsers.find(
-                   queryObject, 
-                   projection
-               ).lean();
+                }
+                // Add the sorting query
+                let sortQuery = { updatedAt: -1 }; // -1 for descending (latest first)
```

Comment:
@VISHNUDAS-tunerlabs let's not do sorting for all queries requested via this function, only for our need we should use.

Reply:
@aks30 added option for that